Homework_2/templates/input_form.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Ввод данных</title>
+</head>
+<body>
+    <h2>Введите имя и электронную почту:</h2>
+    <form action="/submit" method="post">
```

Comment:
Тут больше не нужен атрибут `submit`
```suggestion
    <form method="post">
```